### ISSUE TITLE
How to change default password?

### ISSUE THREAD

**Issue by ghost** [ilyaglow/docker-beef#2]

**Comment by ilyaglow** [comment]
You can mount your own [config.yaml](https://github.com/beefproject/beef/blob/master/config.yaml#L19) inside container and start the image like so:
```sh
docker run -it -p 3000:3000 \
               -p 6789:6789 \
               -p 61985:61985 \
               -p 61986:61986 \
               -v your-config.yaml:/data/config.yaml \
               ilyaglow/beef -c /data/config.yaml
```

**Comment by ilyaglow** [comment]
Also could be implemented inside entrypoint shell script and I will happily accept PR for this feature :)

**Comment by ilyaglow** [comment]
Implemented in 6ddcfadd162b8d56b8761b21232bc638647f9d43
